src/app/components/LiveLabel/index.tsx

```diff
@@ -50,24 +50,36 @@ const LiveLabel = ({
   }
 
   const circle = [
-    [styles.liveLabelCircle, className === 'first-promo' && styles.firstPromo],
+    styles.pulseOuterCircle,
+    className === 'first-promo' && styles.firstPromo,
   ];
 
   return (
     // The id below is a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
     // eslint-disable-next-line jsx-a11y/aria-role
     <span id={id} role="text">
-      <span css={circle}>
-        <span css={styles.livelabelPulse} />
+      <span css={styles.liveLabelContainer}>
+        <span css={circle}>
```

Comment:
I tried this initially but it then only the colour of the pulse would change colour since we removed the colour CSS from the live label text, so I had to wrap them both in a span together in order for it to work. 😩

Reply:
OK cool, thanks for explaining.